### PR TITLE
Chef Infra PR-8765 chocolatey_package updates

### PR DESCRIPTION
**Pull request by kvivek1115**
### Description
Update the `chocolatey_package` doc for extra options added to this resource.

### Issues Resolved
https://github.com/chef/chef/pull/8765

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [ ] All tests pass

Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>

### CODE REVIEW

**Comment by kvivek1115** [comment]
could someone please have a look and let me know further if anything needed from my side?

**Comment by kvivek1115** [comment]
@tas50 PR is good to review and I have matched the added options descriptions.

**Review by tas50 — APPROVED** [review]

File: chef_master/source/resource_chocolatey_package.rst [hunk 1]
```diff
@@ -87,21 +90,50 @@ The chocolatey_package resource has the following properties:
...
+
 ``package_name``
    **Ruby Type:** String, Array
 
    The name of the package. Default value: the name of the resource block.
 
+``user``
+   **Ruby Type:** String
+
+   The username to authenticate feeds.
```
Comment (kagarmoe) [inline chef_master/source/resource_chocolatey_package.rst:111]: Where does the username come from?

File: chef_master/source/resource_chocolatey_package.rst [hunk 2]
```diff
@@ -87,21 +90,50 @@ The chocolatey_package resource has the following properties:
...
+
+   *New in Chef Infra Client 15.3.*
+
+``password``
+   **Ruby Type:** String
+
+   The password to authenticate to the source.
```
Comment (kagarmoe) [inline chef_master/source/resource_chocolatey_package.rst:118]: Along the same lines, where would I have made the password?

**Pull request merged**